### ISSUE TITLE
Syntax error when inserting empty variables

### ISSUE THREAD

**Issue by d-winter** [calebmer/postgraphql#117]
I get the following Error when calling a mutation with all arguments without a value.
As soon as I set at least one variable, the mutation works.

![screen shot 2016-09-20 at 15 51 55](https://cloud.githubusercontent.com/assets/11208517/18672916/38bc4960-7f4a-11e6-8c5b-966f7354ac6b.png)

```
error: syntax error at or near ")"
    at Connection.parseE (/usr/local/lib/node_modules/postgraphql/node_modules/pg/lib/connection.js:539:11)
    at Connection.parseMessage (/usr/local/lib/node_modules/postgraphql/node_modules/pg/lib/connection.js:366:17)
    at Socket.<anonymous> (/usr/local/lib/node_modules/postgraphql/node_modules/pg/lib/connection.js:105:22)
    at emitOne (events.js:96:13)
    at Socket.emit (events.js:188:7)
    at readableAddChunk (_stream_readable.js:177:18)
    at Socket.Readable.push (_stream_readable.js:135:10)
    at TCP.onread (net.js:542:20)
POST / 200 79.240 ms - -
```

**Comment by calebmer** [comment]
Yeah, this definetly looks like a bug, can you look at the Postgres logs and extract the bad query?

This may also be related to https://github.com/calebmer/postgraphql/issues/108

**Comment by calebmer** [comment]
There’s a PR in fixing this, just need to wait for review :+1: https://github.com/calebmer/postgraphql/pull/122

**Comment by calebmer** [comment]
Fix is in `1.9.3` :tada:
